pkg/syncer/cnsoperator/controller/cnsnodevmattachment/cnsnodevmattachment_controller.go

```diff
@@ -340,18 +342,35 @@ func (r *ReconcileCnsNodeVMAttachment) Reconcile(ctx context.Context, request re
 	if instance.DeletionTimestamp != nil {
 		nodeVM, err := dc.GetVirtualMachineByUUID(ctx, nodeUUID, false)
```

Comment:
Could you explain why is GetVirtualMachineByUUID workflow not re-establishing the session on not authenticated error?

Reply:
GetVirtualMachineByUUID is connecting to VC to re-establish the connection. Added vc.Connect to make sure of it.